CMakeLists.txt

```diff
@@ -217,6 +202,22 @@ else()
     message(STATUS "    LIBHWLOC_LIBRARY_DIRS = ${LIBHWLOC_LIBRARY_DIRS}")
 endif()
 
+if(hwloc_targ_SOURCE_DIR)
+    # apply security patch for HWLOC
+    execute_process(
```

Comment:
Look at the options named `ECHO_OUTPUT_VARIABLE, ECHO_ERROR_VARIABLE`.
Generally it's not blocking, but probably you can remove all if blocks below.
Source: https://cmake.org/cmake/help/latest/command/execute_process.html

Reply:
ok but this is enabled in cmake >= 3.18 and I do not want to bump min required ver right now